### ISSUE TITLE
Example request

### ISSUE THREAD

**Issue by bildungsroman** [jerep/gatsby-plugin-rollbar#4]
Could you add any examples of how to actually use this in a gatsby site? I can add it to my config, but I'm not sure how to reference it in my code.

**Comment by Blumed** [comment]
@bildungsroman I don't think you need to. The plugin adds the rollbar script into your `onClientEntry` for you.
